src/main/java/com/google/sps/servlets/MentorApprovalServlet.java

```diff
@@ -76,6 +77,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Update database tables related to mentor approval.
     addApproval(mentorId, approverId);
     addEvidence(isApproved, mentorId);
+
+    // If mentor review is complete, send them a notification.
+    if (Utility.isReviewed(true, mentorId) == Utility.MENTOR_APPROVED) {
+      // If mentor is approved, send notification of type 'approved'.
+      response.setContentType("text/plain");
+      try {
+        request.getRequestDispatcher("/notification?type=approved&modifiedElementId="
+            + mentorId).include(request, response);
+      } catch (ServletException exception) {
+        System.out.println(exception.getMessage());
```

Comment:
use the logger instead of printing so that the errors are better handled

Reply:
Done!